_posts/2024-02-20-moving-to-zulip.markdown

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: "Levelling Up: Zarr Community Transitions to Zulip"
+description: Blog Post on Zarr Community Update → Moving to Zulip
+date: 2024-02-20
+categories: blog
+permalink: /zulip-transition/
+---
+
+Hi, Zarr Community! 👋🏻
+
+We've got an exiciting announcement for you all! 😄
+
+## Community Update: Moving to Zulip 💬
+
+We're excited to announce that the Zarr community has made a significant move
+from [Gitter](https://gitter.im/zarr-developers/community) to [Zulip](https://ossci.zulipchat.com/)
+as our primary chat platform. This transition marks a new chapter for our
+community and offers several advantages for our members.
+
+Join here → <https://ossci.zulipchat.com/>
+
+## Why Zulip? 🤔
+
+Zulip offers a robust and versatile platform for communication and
+collaboration. Its threading model allows for organized and focused
+discussions, making it easier for community members to follow and participate
+in conversations effectively. Additionally, Zulip provides powerful search
+capabilities, ensuring that valuable information shared in the past remains
+accessible to all.
+
+## Hosting Thanks 🙏🏻
+
+We extend our sincere gratitude to the good humans at the [Open Source Science
+Initiative](https://opensource.science/) (OSSCi) for generously hosting the
+Zulip server. Their commitment to supporting open science and collaborative
+research is commendable, and we're thrilled to partner with them on this
+endeavour. 
+
+Shoutout to [Jonathan Starr](https://www.linkedin.com/in/jonathan-starr-b04032284/)
+for helping us! 🙌🏻
+
+## Building a Hub for Open Science 🧬
+
+The OSSCi Zulip server will serve as a hub for various projects under the
```

Comment:
i think it should be "projects in", not "projects under"

Reply:
Done.